### DISCUSSION TITLE
Strategies for Generating Two Non-Replicated Tracks in Predictions and ID Merging

### DISCUSSION THREAD

**Discussion by neugun** [talmolab/sleap#1650] [Help!]
Hi,

I hope this message finds you well. Firstly, I would like to express my gratitude for the excellent tools provided. I am currently using SLEAP on a Windows 10 system with the SLEAP GUI interface.

I've successfully utilized SLEAP to track 2 animals, employing the multi animals top-down ID model for training and then predicting unlabelled frames. However, I've encountered a challenge with the predicted tracks/IDs. Despite all tracks being named as 1 and 2, there are numerous tracks being generated, complicating the analysis (see the attachment). This issue is particularly problematic with analysis software like Simba, as it interprets too many body parts due to the multiple tracks.

I'm seeking advice on the specific reasons for this occurrence and any potential solutions to avoid it. If avoidance is not feasible, could you guide me on how to merge the IDs, specifically consolidating all instances of ID 1 into a single ID 1, and similarly for ID 2?

Your expertise and assistance in resolving this issue would be greatly appreciated.

Thank you for your time and support.

Best regards,
Zhenggang

![image](https://github.com/talmolab/sleap/assets/26133490/29eed3e6-504d-4f5a-8e2c-8ea5e9e75784)

**Comment by Marielamend** [comment]
Hello @neugun!,

Thank you for your inquiry, I have a couple of follow-up questions to be able to best assist you. 
1. What is the order in which you trained your model and ran inference?
2. Do you have multiple projects that have been merged when working on this tracking?
3. Any additional information on your workflow that you can share with us?

Best, 
Mariela

**Comment by roomrys** [comment]
Closing this discussion as it is being followed up in #1652.
